rust_src/src/frames.rs

```diff
@@ -87,6 +88,19 @@ impl LispFrameRef {
     }
 }
 
```

Comment:
As I ponder this a bit, I think all of this could be a `const fn`: https://doc.rust-lang.org/unstable-book/language-features/const-fn.html instead of defining a global variable.